src/content/learn/writing-markup-with-jsx.md

```diff
@@ -122,29 +122,29 @@ img { height: 90px }
 
 </Sandpack>
 
-This is because JSX is stricter and has a few more rules than HTML! If you read the error messages above, they'll guide you to fix the markup, or you can follow the guide below.
+Isso ocorre porque o JSX é mais rígido e possui algumas regras a mais que o HTML! Se você ler as mensagens de erro acima, elas o guiarão para corrigir a marcação ou você poderá seguir o guia abaixo.
 
 <Note>
 
-Most of the time, React's on-screen error messages will help you find where the problem is. Give them a read if you get stuck!
+Na maioria das vezes, as mensagens de erro na tela do React irão ajudá-lo a encontrar onde está o problema. Dê uma lida se você ficar travado!
 
 </Note>
 
-## The Rules of JSX {/*the-rules-of-jsx*/}
+## As Regras do JSX {/*the-rules-of-jsx*/}
 
-### 1. Return a single root element {/*1-return-a-single-root-element*/}
+### 1. Retornar um único elemento raiz {/*1-return-a-single-root-element*/}
 
-To return multiple elements from a component, **wrap them with a single parent tag.**
+Para retornar vários elementos de um componente, **envolva-os com uma única tag pai.**
 
-For example, you can use a `<div>`:
+Por exemplo, você pode usar uma `<div>`:
 
 ```js {1,11}
 <div>
-  <h1>Hedy Lamarr's Todos</h1>
+  <h1>Tarefas do Hedy Lamarr</h1>
   <img 
     src="https://i.imgur.com/yXOvdOSs.jpg" 
     alt="Hedy Lamarr" 
-    class="photo"
+    class="foto"
```

Comment:
```suggestion
    class="photo"
```

Pelas diretrizes, não devemos traduzir nomes de variáveis, classes, etc. No máximo comentários ou textos se for pertinente